### PR TITLE
Submit new NEP-26: Boom: EVM and Protocol Upgrades

### PR DESCRIPTION
**Pull request by VieYang**
## Checklist:

- [x] I have read the **NEP Guideline** document.
- [x] I have checked my NEP Draft is ready.

### CODE REVIEW

**Comment by arisac** [comment]
@VieYang how is the test and implementation plan and progress going?

**Comment by VieYang** [comment]
Please wait until the new code to be confirmed.

File: NEPS/NEP-26.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
+# NEP 26: Boom: EVM and Protocol Upgrades
+
+| Item | Description |
+|:-|:-|
+| NEP | 26 |
+| Title | Boom: EVM and Protocol Upgrades |
+| Author | newchain |
+| Discussions to | https://github.com/newtonproject/NEPs/issues/26 |
+| Status | Draft |
```
Comment (arisac) [inline NEPS/NEP-26.md:10]: if Implemented
please change the status to`Implemented`

File: NEPS/NEP-26.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- Add Blake2 compression function `F` precompile
+- Reduce alt_bn128 precompile gas costs
+- Add ChainID opcode
+- Repricing for trie-size-dependent opcodes
+- Calldata gas cost reduction
+- Rebalance net-metered SSTORE gas cost with consideration of SLOAD gas cost change
+
+This document proposes the following blocks at which to implement these changes in the NewChain networks:
+
+- `17181218` on NewChain testnet
```
Comment (arisac) [inline NEPS/NEP-26.md:37]: also please update to add `timestamp` of implementation

**Review by arisac — APPROVED** [review]
LGTM

**Pull request merged**